Source/config.json

```diff
@@ -28,31 +28,31 @@
       {"_PWSH_Personal_Acces_Token_PAT":"\\$pat\\s*=\\s*['\"][a-z0-9]{52}['\"]"},
       {"_Personal_Acces_Token_PAT":"(\\s|^)[a-z0-9]{52}(\\s|$)"},
       {"_Cloudinary":"cloudinary://.*"},
-      {"_Firebase_URL":"firebaseio\\.com"},
```

Comment:
This needs to be here to escape the .
json parser escapes the \ so the resulting pattern is \. to match firebaseio\.com